src/main/scala/gitbucket/core/service/RepositoryService.scala

```diff
@@ -416,6 +423,30 @@ trait RepositoryService { self: AccountService =>
     }
     .sortBy(_.userName asc).map(t => t.userName -> t.repositoryName).list
 
+  /**
+    * Returns content of template set per repository.
+    *
+    * @param repository the repository information
+    * @param fileBaseName the file basename of template
+    * @return The content of template if the repository has it, otherwise empty string.
+   */
+  def getContentTemplate(repository: RepositoryInfo, fileBaseName: String)(implicit s: Session): String = {
+    val preferred = PluginRegistry().renderableExtensions.map(extension => s"${fileBaseName}.${extension}")
+
+    def choiceTemplate(files: List[FileInfo]): Option[FileInfo] =
+      files.find(f => preferred.contains(f.name)).orElse(files.find(f => f.name.startsWith(fileBaseName)))
```

Comment:
For retrieve template file regardless its extensions, I implemented like that.
But as you said, it is bug what file named start with specific word (like ISSUE_TEMPLATE_XXXXX) is treated as template also.
So, I'll fix to check if the full basename of file is equal to specific word when `.md` and `.markdown` are not found.